integration_tests/pages/arrivalCreate.ts

```diff
@@ -28,8 +28,26 @@ export default class ArrivalCreatePage extends Page {
     cy.get('input[name="expectedDeparture-month"]').type(String(expectedDeparture.getMonth() + 1))
     cy.get('input[name="expectedDeparture-year"]').type(String(expectedDeparture.getFullYear()))
 
-    cy.get('textarea[name="notes"]').type(arrival.notes)
+    cy.get('#conditional-arrived > form >> textarea[name="notes"]').type(arrival.notes)
```

Comment:
It might be easier to use the nunjucks `attributes` option to add `data-cy-*` attributes to make selecting these elements simpler and less brittle. Perhaps even adding `data-cy-arrival` and `data-cy-non-arrival` attributes to the form might be simplest.

Reply:
Yeah this is a good point, I will do the `data-cy` attributes as you suggest